install

```diff
@@ -9,6 +9,13 @@ update_config=2
 binary_arch=
 allow_legacy=
 shells="bash zsh fish"
+config_dir=~/.config/fzf
+fish_dir=~/.config/fish
+if [[ -v XDG_CONFIG_HOME ]] && [ ! -z $XDG_CONFIG_HOME ] && [ -d $XDG_CONFIG_HOME ]; then
```

Comment:
`-z` seems unnecessary. We can write `[[ -v XDG_CONFIG_HOME ]] && [[ -d "$XDG_CONFIG_HOME" ]]`

Reply:
`-z` does seems unnecessary, but the XDG base dir spec allows for that env var to be empty, in which case it should be ignored and code should use a default directory (normally `~/.config`).

From [here](https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html)
>$XDG_CONFIG_HOME defines the base directory relative to which user specific configuration files should be stored. If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

Reply:
Good point, thanks.